src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/MarketStatus.kt

```diff
@@ -26,8 +29,21 @@ data class ExchangesStatusDTO(
     val otc: String? = null
 )
 
+@Serializable
+data class IndicesGroupsStatusDTO(
+    val s_and_p: String? = null,
+    val societe_generale: String? = null,
+    val msci: String? = null,
+    val ftse_russell: String? = null,
+    val mstar: String? = null,
+    val mstarc: String? = null,
+    val cccy: String? = null,
+    val nasdaq: String? = null,
+    val dow_jones: String? = null
+)
```

Comment:
Fields in here should be camelCase w/ the `@SerialName` annotation when necessary for consistency.

Reply:
Fixed.

Reply:
sorry I meant only the fields where the `@SerialName` has to be snake_case need the annotation, single word ones like `nasdaq` don't, but it's fine to keep them.

Reply:
Yeah, that makes sense. My mistake. Fixed.